packages/react-aria-components/docs/FileTrigger.mdx

```diff
@@ -120,6 +120,20 @@ A file trigger can accept multiple files by passsing the `allowsMultiple` proper
 </FileTrigger>
 ```
 
+## Directory Selection
+
+To enable selecting directories instead of files, use the `UNSAFE_directory` property. 
```

Comment:
Since it looks like this is well-supported with browsers we support, I think we can just use `directory`.

Reply:
Fixed by the following commit.

[refac: Remove unsafe prefix](https://github.com/adobe/react-spectrum/pull/5444/commits/10b8984dd21818df134a315c507b010d44184a65)